crates/bifrost/src/loglets/local_loglet/log_store_writer.rs

```diff
@@ -125,7 +131,7 @@ impl LogStoreWriter {
                 .expect("metadata cf exists");
 
             for command in commands {
-                if let Some(data_command) = command.data_update {
+                for data_command in command.data_updates {
```

Comment:
Why can't the `LogStoreWriter` handle the batching of bursty records coming from the invoker?

Reply:
To respect append_batch contract, we want to either commit all records or not. The interface doesn't export partial failures.